.github/workflows/securityAnalysis.js

```diff
@@ -1,12 +1,27 @@
-module.exports = ({core}, path) => {
+module.exports = ({github, core}, path) => {
```

Comment:
Could we change the framing for this to "Reviewed" add-on. I think "trusted" has too may implications